### PR TITLE
[PCFG-89] env management

### PR DESCRIPTION
**Pull request by aomegax**
#### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .devops/code-review-pipelines.yml [hunk 1]
```diff
@@ -5,11 +5,29 @@
...
 # - DANGER_GITHUB_API_TOKEN
 #
+parameters:
+  - name: ENV
+    displayName: Target Environment
+    type: string
+    default: dev
+    values:
+      - dev
+      - uat
```
Comment (pasqualespica) [inline .devops/code-review-pipelines.yml:15]: `prod` missing

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -20,20 +20,32 @@ variables:
...
-    APICONFIG_HOST: "https://api.dev.platform.pagopa.it"
-    APICONFIG_BASEPATH: "/apiconfig/api" # $(APIM_BASEPATH_APICONFIG)
+    AZURE_SUBSCRIPTION: $(DEV_AZURE_SUBSCRIPTION_STORAGE_ACCOUNT)
+    BLOB_CONTAINER_NAME: $(BLOB_CONTAINER_NAME)
+    RESOURCE_GROUP: $(DEV_RESOURCE_GROUP_AZURE)
+    CDN_ENDPOINT: $(DEV_ENDPOINT_AZURE)
+    CDN_PROFILE: $(DEV_PROFILE_NAME_CDN_AZURE)
+    STORAGE_ACCOUNT_NAME: $(DEV_STORAGE_ACCOUNT_NAME)
+    APICONFIG_HOST: $(DEV_APICONFIG_API_HOST)
+    APICONFIG_BASEPATH: $(APIM_BASEPATH_APICONFIG)/v1
```
Comment (pasqualespica) [inline .devops/deploy-pipelines.yml:30]: instead of `v1` hardcoded values, use a variable

File: env.sh [hunk 3]
```diff
@@ -7,6 +7,8 @@ echo "window._env_ = {" >> ./src/env-config.js
 
 # Loop on environment variables prefixed with
 # apiconfig_ and add them to env-config.js
+env | grep -i apiconfig_
```
Comment (pasqualespica) [inline env.sh:10]: remove thi script debug command

File: .devops/deploy-pipelines.yml [hunk 4]
```diff
@@ -146,14 +157,14 @@ stages:
...
 
-#          - task: AzureFileCopy@4
-#            inputs:
-#              sourcePath: '$(Pipeline.Workspace)/Bundle'
-#              azureSubscription: '$(AZURE_SUBSCRIPTION)'
-#              destination: 'azureBlob'
-#              storage: '$(STORAGE_ACCOUNT_NAME)'
-#              containerName: '$(BLOB_CONTAINER_NAME)'
-#            displayName: Deploy to container Blob
+          - task: AzureFileCopy@3
```
Comment (pasqualespica) [inline .devops/deploy-pipelines.yml:160]: with version `3` it works ?
instead of using az cli command ?

**Review by pasqualespica — CHANGES_REQUESTED** [review]

File: .devops/deploy-pipelines.yml (hunk 4 above)
Comment (aomegax) [reply .devops/deploy-pipelines.yml:160]: it's a todo so the task will be commented

**Review by pasqualespica — APPROVED** [review]
🚀

**Pull request merged**